hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsUserSourceImpl.java

```diff
@@ -116,7 +119,7 @@ public MetricsUserSourceImpl(String user, MetricsUserAggregateSourceImpl agg) {
     this.user = user;
     this.registry = agg.getMetricsRegistry();
 
-    this.userNamePrefix = "user_" + user + "_metric_";
+    this.userNamePrefix = "User_" + user + "_metric_";
```

Comment:
this change is ok because all of the existing `newTimeHistogram` calls already capitalize the first letter. So these end up in JMX like `User_foo_metric_...`.  But `newCounter` doesn't do the same capitalization, so the new blockBytesScannedCount ends up `user_foo_metric_blockBytesScannedCount` (lowercase u). So this change here just ensures that all the metrics are similarly capitalized.